src/core/providers/ogr/qgsogrproviderconnection.cpp

```diff
@@ -756,6 +757,53 @@ void QgsOgrProviderConnection::addFieldDomain( const QgsFieldDomain &domain, con
 #endif
 }
 
+void QgsOgrProviderConnection::renameField( const QString &schema, const QString &tableName, const QString &name, const QString &newName ) const
+{
+  checkCapability( Capability::RenameField );
```

Comment:
What is this line actually supposed to do here? shouldn't it be something like if ( !checkCapability() ) return; ?

Reply:
It throws an exception if the capability isn't supported, so there's no need to handle anything manually

Reply:
@nyalldawson , ah, I was wondering, good good.